nearswap/tests/simulation_test.rs

```diff
@@ -3,162 +3,47 @@
 
 #![allow(unused)]
 
-mod ctrtypes;
-use crate::ctrtypes::*;
-
 mod test_utils;
-use crate::test_utils::*;
+use test_utils::*;
+
+use near_sdk_sim::{
+    call, deploy, init_simulator, near_crypto::Signer, to_yocto, view, ContractAccount,
+    UserAccount, STORAGE_AMOUNT,
+};
+use near_sdk_sim::account::AccessKey;
 
 use nearswap::util::*;
 use nearswap::PoolInfo;
-
+use nearswap::NearCLPContract;
```

Comment:
```suggestion
use nearswap::{NearCLPContract, PoolInfo};
```
And remove the line before.

Reply:
ok